services/manage/app/templates/manage/reporting.hbs

```diff
@@ -0,0 +1,160 @@
+          class="btn-warning"
+          disabled=isActionRunning
+          onConfirm=(action "reset")
+        }}
+        <button
+          type="submit"
+          class="btn btn-success ml-1"
+          disabled={{ isActionRunning }}
+          {{action "update"}}
+        >
+          {{entypo-icon "controller-play"}} Execute
+        </button>
+      </div>
+
+      <div class="card-body">
+
+        <div class="table-responsive mt-3">
+          <table class="table table-hover table-sm">
+            <thead>
+              <tr>
+                <th>Publisher</th>
+                <th>Deployment</th>
+                <th class="text-nowrap">Ad Unit</th>
+                <th class="text-nowrap">Ad Unit Size</th>
+                <th>Advertiser</th>
+                <th>Order</th>
+                <th>Line Item</th>
+                <th>Ad</th>
+                <th>Ad Size</th>
+                <th>Impressions</th>
+                <th>Clicks</th>
+                <th>CTR</th>
+              </tr>
+            </thead>
+            <tbody>
+              {{#each model.rows as |item|}}
+                <tr>
+                  <td>
+                    {{#link-to "manage.publishers.edit" item.publisher.id }}
+                      {{ item.publisher.name }}
+                    {{/link-to}}
+                  </td>
+                  <td>
+                    {{#link-to "manage.deployments.edit" item.deployment.id }}
+                      {{ item.deployment.name }}
+                    {{/link-to}}
+                  </td>
+                  <td>
+                    {{#link-to "manage.adunits.edit" item.adunit.id }}
+                      {{ item.adunit.name }}
+                    {{/link-to}}
+                  </td>
+                  <td>{{ item.adunit.size }}</td>
+                  <td>
+                    {{#link-to "manage.advertisers.edit" item.advertiser.id }}
+                      {{ item.advertiser.name }}
+                    {{/link-to}}
+                  </td>
+                  <td>
+                    {{#link-to "manage.orders.edit" item.order.id }}
+                      {{ item.order.name }}
+                    {{/link-to}}
+                  </td>
+                  <td>
+                    {{#link-to "manage.orders.edit.lineitems.edit" item.order.id item.lineitem.id }}
+                      {{ item.lineitem.name }}
+                    {{/link-to}}
+                  </td>
+                  <td>
+                    {{#link-to
+                      "manage.orders.edit.lineitems.edit.ads.edit"
+                      item.order.id
+                      item.lineitem.id
+                      item.ad.id
+                    }}
+                      {{ item.ad.name }}
+                    {{/link-to}}
+                  </td>
+                  <td>{{ item.ad.size }}</td>
+                  <td title="Toal Views: {{ item.impressions }}">{{ number-format item.impressions "0a" }}</td>
```

Comment:
Typo: should be `Total`... also these `title` attrs probably aren't necessary... especially since all the preceding `<td>` elements do not have a title attr.

Reply:
Will fix typo, but the title is there to indicate the non-formatted value aka 10342 instead of 10.3k